LayoutTests/imported/w3c/web-platform-tests/css/cssom/CSSStyleSheet-constructable-baseURL.tentative.html

```diff
@@ -60,7 +60,7 @@
 }, "Constructing sheet with relative URL adds to the constructor document's base URL");
 
 test(() => {
-  assert_throws_dom("NotAllowedError", () => { new CSSStyleSheet({ baseURL: "chrome://"}) });
+  assert_throws_dom("NotAllowedError", () => { new CSSStyleSheet({ baseURL: "https://test:test/"}) });
```

Comment:
Can we file an issue with WPT to get this changed?

Reply:
See the changelog where I said I fixed it upstream via https://github.com/web-platform-tests/wpt/pull/36101 :)